tests/basis/water_energy_B3LYP_631G_input.json

```diff
@@ -25,6 +25,7 @@
   "model": {
     "method": "B3LYP",
     "basis": {
+      "name": "6-31G",
       "description": "6-31G on all Hydrogen and Oxygen atoms",
       "basis_data": {
```

Comment:
Can we rename this to `center_data` and `basis_atom_map` to just `atom_map`. The duplication of names isn't good on fields like this.

Reply:
TODO